tinywl fails to build with Clang due to -Werror

```
$ cc -v
FreeBSD clang version 8.0.0 (branches/release_80 354130) (based on LLVM 8.0.0)
Target: x86_64-unknown-freebsd13.0
Thread model: posix
InstalledDir: /usr/bin

$ gmake -C tinywl
gmake: Entering directory '/path/to/wlroots/tinywl'
cc  \
        -g -Werror -I. \
        -DWLR_USE_UNSTABLE \
        -o tinywl tinywl.c \
        -I/usr/local/include -I/usr/local/include/libdrm -D_THREAD_SAFE -pthread -I/usr/local/include/pixman-1 -L/usr/local/lib -lwlroots  -I/usr/local/include -L/usr/local/lib -lwayland-server  -I/usr/local/include -L/usr/local/lib -lxkbcommon
tinywl.c:260:10: error: 4 enumeration values not handled in switch: 'WLR_INPUT_DEVICE_TOUCH',
      'WLR_INPUT_DEVICE_TABLET_TOOL', 'WLR_INPUT_DEVICE_TABLET_PAD'... [-Werror,-Wswitch]
        switch (device->type) {
                ^
1 error generated.
gmake: *** [Makefile:20: tinywl] Error 1
gmake: Leaving directory '/path/to/wlroots/tinywl'
```

--- a/tinywl/tinywl.c
+++ b/tinywl/tinywl.c
@@ -264,6 +264,8 @@ static void server_new_input(struct wl_listener *listener, void *data) {
 	case WLR_INPUT_DEVICE_POINTER:
 		server_new_pointer(server, device);
 		break;
+	default:
+		break;
 	}
 	/* We need to let the wlr_seat know what our capabilities are, which is
 	 * communiciated to the client. In TinyWL we always have a cursor, even if

